index.html

```diff
@@ -425,6 +425,12 @@ <h2>iss</h2>
       <a data-cite="VC-DATA-MODEL#dfn-verify">verification</a> process.
         </p>
         <p>
+      The value of the <a data-cite="VC-DATA-MODEL#issuer">issuer</a> property can either be a string or an object.
+      If <code>issuer</code> value is a string, <code>iss</code> value MUST match <code>issuer</code> value.
```

Comment:
```suggestion
      If <code>issuer</code> value is a string, and <code>iss</code> is present in the payload or header, the <code>iss</code> value MUST match <code>issuer</code> value.
```

Reply:
`iss` is not a required attribute of the core data model... `issuer` is.

Reply:
```suggestion
      When <code>issuer</code> value is a string, <code>iss</code> value, if present, MUST match <code>issuer</code> value.
```